packages/syft/src/syft/oblv/model.py

```diff
@@ -0,0 +1,441 @@
+# stdlib
+from datetime import datetime
+import json
+import os
+from signal import SIGTERM
+import subprocess
+import sys
+import time
+from typing import Any
+from typing import List
+
+# third party
+from oblv import OblvClient
+import requests
+
+# relative
+from ..core.node.common.exceptions import OblvEnclaveError
+from ..core.node.common.exceptions import OblvUnAuthorizedError
+from ..util import bcolors
+from .oblv_proxy import check_oblv_proxy_installation_status
+
+
+class DeploymentClient:
+
+    # __attr_allowlist__ = ["deployment_id", "user_key_name", "client", "oblv_client"]
```

Comment:
Q: Should this be removed?

Reply:
Solved by [e731947](https://github.com/OpenMined/PySyft/pull/7202/commits/e731947565ee4270f148db04abefb708c97702c4)